core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -25,133 +25,192 @@ THE SOFTWARE.
+                                    <!--TODO convert to "Today", "Tomorrow", "4 days ago", "2 weeks ago", etc. -->
                                 </j:if>
-                            </j:if>
-                            <j:set var="creationDateFormat" value="${%NoCreationDate}" />
-                            <j:set var="creationDateValue" value="${%NoCreationDateValue}" />
-                            <j:if test="${token.creationDate != null}">
-                                <i:formatDate var="creationDateFormat" value="${token.creationDate}" type="both" dateStyle="medium" timeStyle="medium" />
-                                <!--TODO convert to "Today", "Tomorrow", "4 days ago", "2 weeks ago", etc. -->
-                                <j:set var="creationDateValue" value="${%TokenCreation(daysOld)}" />
-                            </j:if>
-                            <span class="token-creation ${oldClazz}" title="${creationDateFormat}">${creationDateValue}</span>
-                            
-                            <span class="to-right">
-                                <j:choose>
-                                    <j:when test="${isStatisticsEnabled}">
-                                        <j:set var="useCounter" value="${token.useCounter}" />
-                                        <j:choose>
-                                            <j:when test="${useCounter > 0}">
-                                                <j:if test="${token.lastUseDate == null}">
-                                                    <!--Should normally never happen without manual XML modification-->
-                                                    <span class="token-use-counter" title="${%TokenNeverUsedTooltip}">
+                                <span class="${oldClazz}" tooltip="${creationDateFormat}">${token.daysAgoString(token.numDaysCreation)}</span>
+                            </td>
+                            <td align="right">${isStatisticsEnabled? token.useCounter :'%N/A'}</td>
+                            <td align="right">
+                                <div class="token-last-used">
+                                    <j:choose>
+                                        <j:when test="${isStatisticsEnabled}">
+                                            <j:set var="useCounter" value="${token.useCounter}" />
+                                            <j:choose>
+                                                <j:when test="${useCounter > 0}">
+                                                    <j:if test="${token.lastUseDate == null}">
+                                                        <!--Should normally never happen without manual XML modification-->
+                                                        <span class="token-use-counter" tooltip="${%TokenNeverUsedTooltip}">
```

Comment:
Not sure if the right line but, the symbol and label have separate tooltips which results in an odd hover effect:


https://github.com/user-attachments/assets/cc9ca336-6dd8-44a7-acd2-e17c5c634002